index.html

```diff
@@ -124,5 +124,32 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-section">
+    <div id="join-hiring">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <form action="#">
+        <button type="submit" id="roles-button">See the roles</button>
+      </form>
+    </div>
+
+    <div id="picture-join">
+      <img id="flying-shoe" src="images/joinbox-flying-shoe.png" alt="flying shoe">
+      <img id="pencil-small" src="images/joinbox-pencil-small.png" alt="pencil small">
+      <img id="green-box" src="images/joinbox-abstract-green-box.png" alt="abstract green box">
+      <img id="blue-ball" src="images/joinbox-abstract-blue-ball.png" alt="abstract blue ball">
+      <img id="flying-febby" src="images/joinbox-flying-febby.png" alt="flying febby">
+    </div>
+
+    <div id="earlyacess-hiring">
+      <h3> Join for early acess </h3>
+      <p> We're gradually opening Ready—get early access to updates and be a part of building the next generation of calendars.</p>
+      <form id="form-hiring">
+        <input type="email" name="email" id="email-hiring" placeholder="Your email">
+        <button>Join</button>
+      </form>
+    </div>
```

Comment:
I don't see which part of an early access has to do with hiring. [See what it's](https://en.wikipedia.org/wiki/Early_access).

There's nothing in this section that hints at hiring anyone (they have that elsewhere). Markup is there to provide the meaning of the content, and the meaning of hiring doesn't align with this content. 

You need to pay more attention to what's going on in the webpage and what's supposed to happen before writing code.